omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -41,175 +51,280 @@ running::
+
+  .. note:: 
+   You can activate the Virtualenv enviornment that we created using::
+
+    $ source ~/Virtual/omero/bin/activate
+
+   This will swtich to using Pip and Python in the Virtualenv directory 
+   :file:`~/Virtual/omero/bin` and any Pip libraries you install, whilst the Virtualenv is activated, 
+   will be installed to :file:`source ~/Virtual/omero/lib`.
+
+  .. note::
+   **(Optional)** To make starting a Virtualenv enviornment easier, 
+   you can add an `alias` to your :file:`.bash_profile`::
 
-    The Homebrew formulae used below provide Python bindings. As
-    described in `Homebrew and Python`_, you should **not** be in an
-    active virtual environment when you ``brew install`` them.
+    alias startVmOmero="source ~/Virtual/omero/bin/activate"
 
-See the :download:`step01_deps.sh <walkthrough/osx/step01_deps.sh>` script for
-the steps described above.
+   Using the command-line terminal, reload your :file:`.bash_profile`::
+
+    $ source ~/.bash_profile
+
+   Now you can activate the Virtualenv enviornment using::
+
+    $ startVmOmero
 
 OMERO installation
 ------------------
 
-OMERO |release| server
-^^^^^^^^^^^^^^^^^^^^^^
+Pre-built server
+^^^^^^^^^^^^^^^^
 
-To install and deploy the |release| release of OMERO.server, run::
+1. Using the command-line terminal, prepare a place for your OMERO server to 
+   be downloaded to. We suggest a folder on your user directory called 'Omero'::
 
-    $ brew install omero53 --with-nginx --with-cpp
-    $ export PYTHONPATH=$(brew --prefix omero53)/lib/python
-    $ export ICE_CONFIG=$(brew --prefix omero53)/etc/ice.config
+    $ mkdir -p ~/Projects/Omero
 
-This will install the OMERO server to /usr/local/Cellar/omero, which means you
-will find the log files in :file:`/usr/local/Cellar/omero/|release|/var/log`.
-The binaries will be linked to :file:`/usr/local/bin`::
+   Run the following command to download a build of OMERO.Server::
 
-    $ which omero
-    /usr/local/bin/omero
+    $ curl https://downloads.openmicroscopy.org/omero/5.3.4/artifacts/OMERO.server-5.3.4-ice36-b69.zip > ~/Projects/Omero/server.zip
 
-Install Ice 3.6 extension for Python and OMERO python dependencies::
+   Extract the :file:`server.zip`
 
-    $ pip install -r $(brew --prefix omero53)/share/web/requirements-py27-all.txt
-    $ cd /usr/local
-    $ bash bin/omero_python_deps
+2. Once extracted, open your :file:`.bash_profile` in a text editor, 
+   such as the built-in TextEdit app::
 
-Start database server::
+    $ open -a TextEdit.app ~/.bash_profile
 
-    $ pg_ctl -D /usr/local/var/postgres -l /usr/local/var/postgres/server.log -w start
+   Add an enviornment variable :envvar:OMERO_SERVER to the :file:`.bash_profile` which points
+   to the location of the OMERO executabale::
 
-Create database and user::
+    # OMERO Server distribution directory
+    export OMERO_SERVER=~/Projects/Omero/server
```

Comment:
This does not work as written, as there is no subdirectory called ``/server``. Maybe write in the above text "Replace the OMERO.server.xxx with the name of your unzipped OMERO.server directory." and exchenge the ``server`` -> ``OMERO.serverxxx`` here

Reply:
Which does not work for me anyway. So this needs one more revision.